package.json

```diff
@@ -57,10 +57,11 @@
     "serve:client": "cd ./client && npm run serve",
     "start": "npm-run-all ensure-env -p develop:server serve:client",
     "test": "npm-run-all -p test:*",
-    "test:client": "cd ./client && npm test && cd ../",
-    "test:curriculum": "cd ./curriculum && npm test && cd ../",
+    "test:client": "cd ./client && npm test",
+    "test:curriculum": "cd ./curriculum && npm test",
+    "test-curriculum-full-output": "cd ./curriculum && npm run test:full-output",
```

Comment:
Is there a particular reason to leave off the escape from the directory: `&& cd ../` ?

Reply:
Yep, it didn't seem to be necessary.  When I ran the scripts without it they didn't change my dir.

Like if you create the following bash script

```sh
cd dir
pwd
```

the `./dir` directory gets output, but you end up in `./`

Reply:
Ah right. I would have never noticed.